### PR TITLE
Feature: tag cloud grid view

### PR DESCRIPTION
**Pull request by vadymmarkov**
# Why?

This new component will be used on the "explore" screen, but could potentially be re-used in other places.

# What?

- Add `TagCloudGridView` with demo and snapshot tests
- Drop iOS12 support

# Version Change

Breaking change because we're dropping iOS12 support.

# UI Changes

Simulator Screen Shot - iPhone 11 Pro - 2021-06-07 at 15 26 17

### CODE REVIEW

File: FinnUI.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'FinnUI'
-  s.version      = '22.3.0'
+  s.version      = '23.0.0'
   s.summary      = "FINN's iOS UI Features"
   s.author       = 'FINN.no'
   s.homepage     = 'https://schibsted.frontify.com/d/oCLrx0cypXJM/design-system'
```
Comment (ninarg) [inline FinnUI.podspec:6]: We should update the podspec to iOS 13 as well 😄

Comment (vadymmarkov) [reply FinnUI.podspec:6]: Nice catch 😉

File: Sources/Components/TagCloudGridView/Cell/TagCloudCell.swift [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+    // MARK: - Internal properties
+
+    weak var remoteImageViewDataSource: RemoteImageViewDataSource? {
+        didSet {
+            iconView.dataSource = remoteImageViewDataSource
+        }
+    }
+
+    private var showShadow = false
```
Comment (ninarg) [inline Sources/Components/TagCloudGridView/Cell/TagCloudCell.swift:33]: move ⬇️  to private properties

**Review by ninarg — APPROVED** [review]
Looks amazing! 🥇

**Pull request merged**